From e94de3aad89eecac7eaf8236b4c0b2c7919bc70b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Fri, 27 Oct 2017 04:19:30 +0200
Subject: [PATCH] pasteManager: Remove excess arguments

Our helper method takes less arguments than the GTK+ method it's based
on - remove the excess argument to avoid warnings.

Fixes https://gitlab.gnome.org/GNOME/polari/issues/14
---
 src/pasteManager.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/pasteManager.js b/src/pasteManager.js
index e39f493..aa321f4 100644
--- a/src/pasteManager.js
+++ b/src/pasteManager.js
@@ -122,7 +122,7 @@ var DropTargetIface = new Lang.Interface({
         if (!this.can_drop)
             return Gdk.EVENT_PROPAGATE;
 
-        if (!Polari.drag_dest_supports_target(widget, context, null))
+        if (!Polari.drag_dest_supports_target(widget, context))
             return Gdk.EVENT_PROPAGATE;
 
         Polari.drag_dest_request_data(widget, context, time);
@@ -138,7 +138,7 @@ var DropTargetIface = new Lang.Interface({
         if (!this.can_drop)
             return Gdk.EVENT_PROPAGATE;
 
-        if (!Polari.drag_dest_supports_target(widget, context, null))
+        if (!Polari.drag_dest_supports_target(widget, context))
             return Gdk.EVENT_PROPAGATE;
 
         let info = Polari.drag_dest_find_target(widget, context);
-- 
2.17.0

